web/pages/group/[...slugs]/index.tsx

```diff
@@ -150,9 +154,14 @@ export function GroupPageContent(props: { groupParams?: GroupParams }) {
   const { groupParams } = props
   const router = useRouter()
   const { slugs } = router.query as { slugs: string[] }
-  const page = slugs?.[1] as typeof groupSubpages[number]
+  // const page = slugs?.[1] as typeof groupSubpages[number]
+
+  const [tab, setTab] = usePersistentState('questions', {
```

Comment:
Not sure why we can't just include the second slug as the start state, so you can begin on a tab.